backends/models.py

```diff
@@ -0,0 +1,35 @@
+"""
+OAuth token model
+"""
+import datetime
+
+import pytz
+from django.db import models
+from django.db.models import TextField, CharField, DateTimeField
+
+
+class OAuthToken(models.Model):
+    """
+    Store OAuth access and refresh tokens for an external API, such as FluidReview,
+    that has short-lived refresh tokens (invalidated whenever used).
+    """
+    api = CharField(max_length=30, unique=True, primary_key=True, null=False)
+    access_token = TextField(null=True)
+    refresh_token = TextField(null=True)
+    token_type = CharField(max_length=30, default='Bearer')
+    expires_on = DateTimeField(null=False)
+
+    @property
+    def json(self):
+        """
+        Return the token in an OAuth2Session-compatible format
+
+        Returns:
+            dict: Token attributes
+        """
+        return {
+            'access_token': self.access_token,
+            'refresh_token': self.refresh_token,
+            'token_type': self.token_type,
+            'expires_in': int((self.expires_on - datetime.datetime.now(tz=pytz.UTC)).total_seconds())
```

Comment:
Can you use the `utc_now` function you defined in `utils`?

Reply:
Sure